web/modules/custom/hoeringsportal_audit_log/src/Helpers/ConfigHelper.php

```diff
@@ -87,13 +88,39 @@ public function escapeProviderId(string $input) {
   }
 
   /**
-   * Get route names.
+   * Get routes to audit from config.
    *
    * @return array<string, string>
-   *   Array of route names.
+   *   routes to audit.
+   */
+  private function getRoutesToAudit() {
+    try {
+      $routesToAudit = Yaml::decode($this->moduleConfig->get('routes_to_audit'));
+      return $routesToAudit;
```

Comment:
You may want to make sure that you actually return an array:

```suggestion
      return (array) $routesToAudit;
```